packages/protocol/contracts/governance/Governance.sol

```diff
@@ -629,7 +629,7 @@ contract Governance is
     uint256 weight = getLockedGold().getAccountTotalLockedGold(account);
     require(proposal.isApproved(), "Proposal not approved");
     require(stage == Proposals.Stage.Referendum, "Incorrect proposal state");
-    require(value != Proposals.VoteValue.None, "Vote value unset");
+    require(value > Proposals.VoteValue.Abstain, "Vote value unset");
```

Comment:
```suggestion
    require(value <= Proposals.VoteValue.Abstain, "Vote value unset");
```

Reply:
I am bit confused. I will make this change and see how this works. Now, I understand the reason for this change. Thank you.

Reply:
@yorhodes This change will fail most of the test cases related to vote method as 'yes' and 'no' are greater then 'abstain'. I am not clear on this change. Please confirm if this change is correct. Thank you.